src/App.tsx

```diff
@@ -10,33 +12,22 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
-  return (
-    <div className="section">
-      <h1 className="title">
-        Selected tab is Tab 1
-      </h1>
-
-      <div data-cy="TabsComponent">
-        <div className="tabs is-boxed">
-          <ul>
-            <li className="is-active" data-cy="Tab">
-              <a href="#tab-1" data-cy="TabLink">Tab 1</a>
-            </li>
+  const [selectedTabId, setSelectedTabId] = useState('tab-1');
```

Comment:
```suggestion
  const [selectedTabId, setSelectedTabId] = useState<Tab>('tab-1');
```

Reply:
I used useState<string>('tab-1') to specify that selectedTabId is of type string. It's the correct approach since selectedTabId is intended to store a string identifier, and below, I have the handleTabSelected method, which takes an argument of type Tab for handling tab selection, ensuring type consistency.